packages/libs/react-ui/src/components/SelectField/index.ts

```diff
@@ -0,0 +1 @@
+export { type ISelectFieldProps, SelectField } from './SelectField';
```

Comment:
I think the linter should split these now?

Reply:
hmm I'm not sure. I don't see any errors so maybe this case isn't covered?

Reply:
I think it makes sense to have it work for exports too, but that would need to be a separate PR. I'll update this specific case for this PR